### ISSUE TITLE
misleading description of parameter `p` in `mass_absolute`

### ISSUE THREAD

**Issue by NimaSarajpoor** [TDAmeritrade/stumpy#563]
I found this before, and not sure if it is a mistake or not.

In `core.mass`, the description of parameter `p` is as follows:
```
    p : float, default 2.0
        The p-norm to apply for computing the Minkowski distance. This parameter is
        ignored when `normalize == False`.
```

So, I assume `p` is only for normalized version. However, we see the following description of the same parameter in `core.mass_absolute`:

```
p : float, default 2.0
        The p-norm to apply for computing the Minkowski distance.
```

Maybe we should remove parameter `p` here in non-normalized version, or add a note to let user know it has not been implemented yet.

**Comment by seanlaw** [comment]
@NimaSarajpoor No, I think there is a mistake and it should read :

```This parameter is ignored when `normalize==True````

Oops! Would you mind finding all instances of `normalize == False` (for `p`) and changing it to `normalize == True`?

**Comment by NimaSarajpoor** [comment]
@seanlaw 
That is a very good news for me :) Because I was thinking of trying other parameter `p` for my case (which is non-normalized) and see what happens... and then I saw the docstring and thought I could not use it... 

I will take care of it.

**Comment by seanlaw** [comment]
@NimaSarajpoor Thank you!

**Comment by seanlaw** [comment]
Closed in #565
